docs/gen-ai/openai.md

```diff
@@ -0,0 +1,162 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: OpenAI traces and metrics
+--->
+
+# Semantic Conventions for OpenAI operations
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [OpenAI Span attributes](#openai-span-attributes)
+- [OpenAI Metric attributes](#openai-metric-attributes)
+
+<!-- tocstop -->
+
+A request to an Generative AI is modeled as a span in a trace.
+
+**Span kind:** MUST always be `CLIENT`.
+
+The Semantic Conventions for [OpenAI](https://openai.com/) extend and override the semantic conventions
+for [Gen AI Spans](gen-ai-spans.md) and [Gen AI Metrics](gen-ai-metrics.md).
+
+`gen_ai.system` MUST be set to `"openai"`.
+
+## OpenAI Span attributes
+
+These attributes track input data and metadata for a request to an OpenAI model. The attributes include general Generate AI
+attributes and ones specific the OpenAI.
+
+<!-- semconv trace.gen_ai.openai.client -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`gen_ai.operation.name`](/docs/attributes-registry/gen-ai.md) | string | The name of the operation being performed. [1] | `chat`; `text_completion` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`gen_ai.request.model`](/docs/attributes-registry/gen-ai.md) | string | The name of the GenAI model a request is being made to. [2] | `gpt-4` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`gen_ai.system`](/docs/attributes-registry/gen-ai.md) | string | The Generative AI product as identified by the client or server instrumentation. [3] | `openai` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`error.type`](/docs/attributes-registry/error.md) | string | Describes a class of error the operation ended with. [4] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | `Conditionally Required` if the operation ended in an error | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| [`gen_ai.openai.request.json_schema.name`](/docs/attributes-registry/gen-ai.md) | string | The name of the JSON schema used for the response. | `chat_response` | `Conditionally Required` [5] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
do we need this attribute? the only concern is potential bloating of telemetry - every time we add a new attribute we should try to balance the usefulness of it vs potential cost of populating it. 

If we don't know the scenario when it could be useful, let's not add it for now